builder/oci/config_test.go

```diff
@@ -129,6 +129,17 @@ func TestConfig(t *testing.T) {
 		}
 	})
 
+	t.Run("NicAttachmentType", func(t *testing.T) {
```

Comment:
Looking at that documentation the argument has a set of valid value. But I don't see a check for these values within the `config#Prepare` method. Should there a be a validation check to prevent builds from failing during runtime if a unsupported value is supplied? 

From the code it is not clear to me what happens if an invalid option is suppled.

Reply:
Added a check for valid values per your suggestion to config.io